### ISSUE TITLE
Zip code error on map

### ISSUE THREAD

**Issue by FivelFred** [justarrived/just-match-frontend#627]
On production, as a user who is not logged in the map puts the job in Demark. Turns out it sets the pin on a street called Sweden. Needs to be fixed ASAP. 

![post zip code error ends up in denmark](https://cloud.githubusercontent.com/assets/16607246/16201023/fad8da8c-370f-11e6-890c-ac4a5736f69c.PNG)

**Comment by FivelFred** [comment]
Not sure if this is back-end or front-end issue?

**Comment by ti-sedp** [comment]
I plotted makers from zip-latitude and zip-longitude field got it from api

![2559-06-20_23h21_25](https://cloud.githubusercontent.com/assets/16758634/16201969/eaea5712-373d-11e6-8b1b-34890141afe0.png)

**Comment by buren** [comment]
Hm.. I need to investigate this a bit further. I tried one of the jobs in production and it displayed the job in Denmark, but when checking the coordinates I got some place outside of Kiruna (Sweden) ❓ 

I'm not doing anything fancy, I simply take the address fields and send them to Google and ask for the latitude, longitude coordinates.. I saw that I'm using a `float` type in the database to store these coordinates, so perhaps there are some rounding issues that causes this. I'll look into it ASAP.
